lib/template.html

```diff
@@ -38,7 +38,12 @@
             playerBrandingId: "pleaseFillTheRightPlayerId",
             skin: {
                 inline: {
+                    startScreen: {
+						showDescription: false,
```

Comment:
Please fix the spacing over here.

Reply:
I have been asking this, Do we really need to hide the title and description everywhere with receiver? I don't remember anyone asking for hiding title and description.

Reply:
Aldi, if you want to change the parameters that are delivered in default receiver then please open the task and we will place any parameters that are required. Current parameters come to template long time ago and Anton is not responcible for adding them. I am not even sure that it was added by my someone from my team so I suppose we need to leave it as it at least in this ticket.